Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -0,0 +1,82 @@
+"""This contains the plotting and geo data management methods for the COVIDcast signals."""
+
+import os
+
+import geopandas as gpd
+import pandas as pd
+
+from covidcast import covidcast
+
+SHAPEFILE_PATHS = {"county": "../shapefiles/county/cb_2019_us_county_5m.shp",
+                   "state": "../shapefiles/state/cb_2019_us_state_5m.shp"}
+
+CURRENT_PATH = os.path.dirname(os.path.realpath(__file__))
+
+
+def get_geo_df(data: pd.DataFrame,
+               geo_value_col: str = "geo_value",
+               geo_type_col: str = "geo_type") -> gpd.GeoDataFrame:
+    """Append polygons to a dataframe for a given geography.
+
+    Default values correspond to return of
+    covidcast.signal(). Currently only supports counties and states.
+
+    :param data: DataFrame of values and geographies
+    :param geo_value_col: name of column containing values of interest
+    :param geo_type_col: name of column containing geography type
```

Comment:
For the docs, it'd be good for this docstring to explain what a GeoDF is and what package it comes from, and what it means to "append polygons to a dataframe". That is, it should link to geopandas, say explicitly that there will be a new column with a certain name, say that the column is the geometry of each location, etc.

Reply:
Added some more info, let me know what you think